liota/core/package_manager.py

```diff
@@ -288,8 +288,20 @@ def _cmd_handler_list(self, parameter):
     #-----------------------------------------------------------------------
     # This method is used to handle statistical commands
 
-    def _cmd_handler_stat(self, parameter):
-        if parameter == "metrics" or parameter == "met":
+    def _cmd_handler_stat(self, parameters):
+        if parameters[0] == "package" or parameters[0] == "pkg":
```

Comment:
If we are just logging it, is the command useful? Anyway we are getting this information from the logs by list pkg. I think the  purpose was to get the output to Pulse Agent, which is not met.

Reply:
This command can let u directly query whether that pkg is loaded or not. It's more straightforward.

Reply:
Ok, considering that once we start returning the output to the user who runs liotapkg.sh some time in future, this command will be helpful.